mdl/src/main/herd/scripts/installShepherd.sh

```diff
@@ -46,6 +46,7 @@ fi
 execute_cmd "echo \"From $0\""
 # Download and serve herd-ui (shepherd)
 execute_cmd "mkdir -p ${deployLocation}/herd-ui"
+#TODO faild sometimes, need retry on wget
```

Comment:
wget has implicit retries, did you want to add additional retry logic around this?